.travis.yml

```diff
@@ -221,6 +221,7 @@ jobs:
     if: *deploy-if
     env: *github-env
     script: skip
+    os: osx
```

Comment:
That is all it takes? \o/

Reply:
I think so... it's hard to test, though, so we'll find out next time I cut a release!